README.md

```diff
@@ -13,7 +13,7 @@ yarn
 In order to inspect only the transaction to be refunded, you can run(error is expected in the output):
 
 ```
-export INFURA_KEY=
+export NODE_URL=
 export NETWORK=mainnet
```

Comment:
The network is now only used for printing a link to the mined tx which doesn't even work properly for some networks. I think we can drop the `NETWORK` env and simply print the `txHash` and tell people to look it up in the respective block explorer.